exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/grpc/DefaultGrpcExporter.java

```diff
@@ -89,7 +89,7 @@ public void onFailure(Throwable t) {
                         + "This usually means that your collector is not configured with an otlp "
                         + "receiver in the \"pipelines\" section of the configuration. "
                         + "Full error message: "
-                        + t.getMessage());
```

Comment:
why get rid of the throwable message here?

Reply:
I don't think it was intended - the throwable is only wrapping Status, so it's message has some extra boilerplate that causes it not to match the other implementation with no extra information.